content/how-to/account-management/_index.md

```diff
@@ -16,10 +16,10 @@ managing invoices, and more.
 
 ### Account Management
 
-- [Change an account's company name](/support/how-to/change-account-company-name)
+- [Change your Account Company Name](/docs/portal-onboarding-guide/administrative/change_account_company_name#)
 - [Change account information](/support/how-to/change-account-information/)
-- [Change the account owner](/support/how-to/change-account-owner/)
-- [Change the account's primary contact](/support/how-to/change-to-new-primary-contact/)
+- [Change your Cloud Account Owner](/docs/portal-onboarding-guide/administrative/change_cloud_account_owner)
```

Comment:
keep old URL here, but change the URL in the `_redirects` file

Reply:
I have updated the _redirect file, but getting "conflict"